release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2775,6 +2775,53 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, when `openshift-sdn` pods were deployed during the {product-title} upgrading process, the Open vSwitch (OVS) storage table was cleared. This issue occurred on {product-title} {product-version}.19 and later versions. Ports for existing pods had to be re-created and this disrupted numerous services. With this release, a fix ensures that the OVS tables do not get cleared and pods do not get disconnected during a cluster upgrade operation. (link:https://issues.redhat.com/browse/OCPBUGS-45955[*OCPBUGS-45955*])
+
+* Previously, you could not remove a finally pipeline task from the *edit Pipeline* form if you created a pipeline with only one finally task. With this release, you can remove the finally task from the *edit Pipeline* form and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-45950[*OCPBUGS-45950*])
+
+* Previously, the `aws-sdk-go-v2` software development kit (SDK) failed to authenticate an `AssumeRoleWithWebIdentity` API operation on an {aws-first} {sts-first} cluster. With this release, `pod-identity-webhook` now includes a default region, and this issue no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-45940[*OCPBUGS-45940*])
+
+* Previously, the installation program populated the `network.devices`, `template` and `workspace` fields in the `spec.template.spec.providerSpec.value` section of the {vmw-full} control plane machine set custom resource (CR).
+These fields should be set in the {vmw-short} failure domain, and the installation program populating them caused unintended behaviors.
+Updating these fields did not trigger an update to the control plane machines, and these fields were cleared when the control plane machine set was deleted.
+With this release, the installation program is updated to no longer populate values that are included in the failure domain configuration.
+If these values are not defined in a failure domain configuration, for instance on a cluster that is updated to {product-title} {product-version} from an earlier version, the values defined by the installation program are used.
+(link:https://issues.redhat.com/browse/OCPBUGS-45839[*OCPBUGS-45839*], link:https://issues.redhat.com/browse/OCPBUGS-37064[*OCPBUGS-37064*])
+
+* Previously, the ClusterTasks were listed on the *Pipelines builder* and *ClusterTask list* pages in the *Tasks* navigation menu. With this release, the ClusterTasks are deprecated from Pipelines 1.17 and the ClusterTask dependency is removed from static plug-in. On the Pipelines builder page, you will only see the present task in the Namespace and Community tasks. (link:https://issues.redhat.com/browse/OCPBUGS-45248[*OCPBUGS-45248*])
```

Comment:
Based on use in https://docs.openshift.com/pipelines/1.17/create/remote-pipelines-tasks-resolvers.html
```suggestion
* Previously, the ClusterTasks were listed on the *Pipelines builder* and *ClusterTask list* pages in the *Tasks* navigation menu. With this release, the `ClusterTask` functionality is deprecated from Pipelines 1.17 and the `ClusterTask` dependency is removed from static plug-in. On the Pipelines builder page, you will only see the present task in the Namespace and Community tasks. (link:https://issues.redhat.com/browse/OCPBUGS-45248[*OCPBUGS-45248*])
```